### DISCUSSION TITLE
v1.0.0 GA Release

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/actions#1088] [Announcements]
This is the first GA release for the **GitVersion** and **GitReleaseManager** GitHub actions and Azure DevOps tasks.

In this release the main focus was to add the possibility to install and run the **GitVersion** and **GitReleaseManager** tools from the **GitTools** organization in both GitHub actions workflows as well as in the Azure DevOps pipelines.

The documentation on how to use these actions/tasks can be found in the docs folder, and real example of workflows/pipelines can be found in the `.azure` and `.github` folders

<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/actions/releases/tag/v1.0.0'>v1.0.0 GA Release</a>.</em>

**Comment by samsmithnz** [comment]
Congrats! Big milestone for a really useful tool!

**Reply by arturcic** [reply]
Thank you, we needed to get it to a GA state, and focus on v2.0.0 where we will refactor the code to improve the maintainability and testability

**Reply by samsmithnz** [reply]
I'm curious, do you publish a roadmap somewhere? I don't see one in the issues or main readme.

**Reply by arturcic** [reply]
Not at the moment as it was as a side project for the main GitVersion and GitReleaseManager projects.
I will try to improve that with time and add milestones with the issues I want to work on.
The focus for v2 will be the refactoring, the updates necessary for upcomming GitVersion v6 GA release, and updates for GitReleaseManager that @gep13 will suggest
